aspnetcore/security/authentication/azure-active-directory/index.md

```diff
@@ -17,3 +17,11 @@ uid: security/authentication/azure-active-directory/index
 * [Calling a Web API in an ASP.NET Core Web Application Using Azure AD](https://azure.microsoft.com/documentation/samples/active-directory-dotnet-webapp-webapi-openidconnect-aspnetcore/)
```

Comment:
Arguably the articles are really the manifestation of samples (for instance for the first one: https://github.com/Azure-Samples/active-directory-dotnet-webapp-openidconnect-aspnetcore)